### ISSUE TITLE
Build fail, missing jsQR

### ISSUE THREAD

**Issue by chevdor** [paritytech/parity#7948]
> I'm running:
>
> - **Which Parity version?**: try to build master so 5b4abec2dbc3f5653b2caf2cc481411a29c7d00b

> - **Which operating system?**: Windows / **MacOS** / Linux
> - **How installed?**: via installer / homebrew / binaries / **from source**
> - **Are you fully synchronized?**: no / **yes**
> - **Which network are you connected to?**: **ethereum** / ropsten / kovan / ...
> - **Did you try to restart the node?**: no / yes => n/a

I am trying to build parity but the UI seems to ref. https://github.com/JodusNodus/jsQR.git which is an invalid or maybe private repo.

```
error: failed to run custom build command for `parity-ui-old-dev v1.9.0 (https://github.com/parity-js/dapp-wallet.git?rev=1a58bf4836c84e1632e27ef607b5a388abd2bf2d#1a58bf48)`
process didn't exit successfully: `/Users/will/projects/ethereum/paritytech/parity/target/release/build/parity-ui-old-dev-a38edfe530e8669d/build-script-build` (exit code: 101)
--- stderr
npm ERR! Error while executing:
npm ERR! /usr/local/bin/git ls-remote -h -t https://github.com/JodusNodus/jsQR.git
npm ERR! 
npm ERR! remote: Invalid username or password.
npm ERR! fatal: Authentication failed for 'https://github.com/JodusNodus/jsQR.git/'
npm ERR! 
npm ERR! exited with error code: 128

```

**Comment by JodusNodus** [comment]
The older system used here still relied on a fork of mine. I removed it without thinking about supporting the older versions. Forked it again so it should work now.

**Comment by chevdor** [comment]
Thanks @JodusNodus, it is now possible to build again.
I opened #7949 for the discussion whether or not to switch to cozmo/jsQR as dependency. That would let you delete your fork and prevent further issues.

**Comment by 5chdn** [comment]
Fixed in 1.9.3 and 1.8.10
